pkgs/cronet_http/lib/src/cronet_client.dart

```diff
@@ -332,6 +332,7 @@ class CronetClient extends BaseClient {
       builder.setUploadDataProvider(
           jb.UploadDataProviders.create2(body.toJByteBuffer()), _executor);
     }
+    builder.setHttpMethod(request.method.toJString());
```

Comment:
Weird this is not caught in an existing test.

@brianquinlan ?

Reply:
Because I accidentally deleted the test while also breaking the code :-(